ci/infra/testrunner/tests/test_deployments.py

```diff
@@ -1,21 +1,26 @@
-from skuba import Skuba
-from kubectl import Kubectl
 import pytest
-import time
-from timeout_decorator import timeout
-
-def test_nginx_deployment(setup, skuba, kubectl):
-    skuba.node_join(role="worker", nr=0)
-    skuba.node_join(role="worker", nr=1)
-    masters = skuba.num_of_nodes("master")
-    workers = skuba.num_of_nodes("worker")
-    assert masters == 1
-    assert workers == 2
-    kubectl.create_deployment("nginx", "nginx:stable-alpine")
-    kubectl.scale_deployment("nginx", workers)
-    kubectl.expose_deployment("nginx", 80)
-    kubectl.wait_deployment("nginx", 3)
-    assert kubectl.count_available_replicas("nginx") == workers
-    result = kubectl.test_service("nginx")
-    assert "Welcome to nginx" in result
+import json
+import requests
+
+from kubectl import Kubectl
+
+
+def test_nginx_deployment(deployment, kubectl):
+    workers = kubectl.skuba.num_of_nodes("worker")
+    kubectl.run_kubectl("create deployment nginx --image=nginx:stable-alpine")
```

Comment:
Maybe add a test making sure the nginx deployment is successfully deployed before you try to scale it up. For example as as described [here](https://kubernetes.io/docs/concepts/services-networking/connect-applications-service/) and then `ssh sles@nodeip curl podIP:podPort` to verify nginx is working internally and it's reachable within the cluster. This is not necessary though, however if your external service test fails, you would wish to have the internal one (as debugging step).

Reply:
Maybe to add a more checks, but I already check that the service is running, in the last assert from line 23 - 26

Reply:
yes, this can be done later as well, that's why I gave you already my blessing to merge this :) :P